README.md

```diff
@@ -1,5 +1,102 @@
 # How to run
+Run ```docker run -p 5432:5432 -e POSTGRES_PASSWORD=password -e POSTGRES_USER=user postgres``` before starting
+the main Spring Boot application. You can start it using the run button on IntelliJ or in another terminal 
+run ``mvn clean package`` then run ``java -jar target/spring-boot-example-0.0.1-SNAPSHOT.jar server application.yml
+``
+# What is the application?
+REST API monolith Spring application that uses a postgres database to store customers. A REST API is exposed to the user. 
 
-Run ```docker run -p 5432:5432 -e POSTGRES_PASSWORD=password -e POSTGRES_USER=user postgres``` before starting 
-the main Spring Boot application.
+### Endpoints
 
+1. **Get all customers**
+    - **Request:**
+        - Method: GET
+        - URL: `/api/v1/customers`
+        - Example Request:
+          ```
+          GET /api/v1/customers
+          ```
+
+    - **Response:**
+        - Status Code: 200 OK
+        - Body: Array of customer objects containing `id`, `name`, `email`, and `age` fields.
+
+2. **Add a new customer**
```

Comment:
You're missing a test for this. Modify the endpoint so it returns the customer and test it accordingly.